libraries/AP_DroneCAN/AP_DroneCAN_DNA_Server.h

```diff
@@ -136,6 +136,10 @@ class AP_DroneCAN_DNA_Server
 
     //Run through the list of seen node ids for verification
     void verify_nodes();
+        
+    // Log NodeInfo
+    void log_NodeStatus(uint8_t node_id, uint32_t uptime_sec, uint8_t healthy, uint8_t mode, uint32_t delta);
+    uint32_t last_can_init_delta_ms[126];
```

Comment:
this is using a lot of memory for users with the option enabled. I think we should dynamically allocated it, so declare as a pointer here
variable name is also a bit confusing, last_node_status_ms[] would be better